pikaur/print_department.py

```diff
@@ -667,3 +670,21 @@ def get_aur_sort_key(pkg: AURPackageInfo) -> Tuple[AurSortKey, str]:
             ))
             print(format_paragraph(f'{package.desc}'))
     return sorted_packages
+
+
+def _get_total_size(
```

Comment:
it should be a method of InstallInfoFetcher

Reply:
updated

Reply:
thanks!